functions/Get-DbaJobOutputFile.ps1

```diff
@@ -0,0 +1,148 @@
+﻿Function Get-DbaJobOutputFile
```

Comment:
Since there are more types of jobs, not in the least PowerShell Jobs, this function name should clarify it's about AgentJobs, so I propose Get-DbaAgentJobOutputFile